tests/build_parents_file.sh

```diff
@@ -0,0 +1,251 @@
+#!/bin/bash
```

Comment:
Minor but not sure name-wise if `build_parents_file` includes all the functionality as we also getting all children for the parents.

Reply:
@thepetk The name is referring to the `parents.yaml` file this script builds rather than referring to everything it does to build the file, though I'm open to suggestions to improve the descriptor.

Reply:
Yeah makes sense. TBH I don't have any good suggestions atm so let's keep it this way and we can revisit in the future if necessary :)